src/main/java/uk/gov/justice/hmpps/prison/service/SentenceEnvelopeService.kt

```diff
@@ -47,26 +39,22 @@ class SentenceEnvelopeService(
         pageRequest,
       )
     val activeBookings =
-      offenderBookingRepository.findAllByBookingIdIn(bookingIds.stream().map(OffenderBookingId::bookingId).toList())
-    val calculableSentenceEnvelopes = activeBookings.stream()
-      .map { offenderBooking: OffenderBooking -> determineCalculableSentenceEnvelope(offenderBooking) }
-      .toList()
+      offenderBookingRepository.findAllByBookingIdIn(bookingIds.map(OffenderBookingId::bookingId).toList())
+    val calculableSentenceEnvelopes =
+      activeBookings.map { determineCalculableSentenceEnvelope(it) }.sortedBy { it.bookingId }
```

Comment:
had to add a `sort` here so that we get the results back in the same order we did previously